tests/unocss.test.ts

```diff
@@ -1,14 +1,51 @@
 import { assertSiteSnapshot, build, getSite } from "./utils.ts";
-import postcss from "../plugins/postcss.ts";
 import unocss from "../plugins/unocss.ts";
+// https://github.com/denoland/deno/issues/19096
+import transformerVariantGroupImport from "npm:@unocss/transformer-variant-group@0.57.7";
+import transformerDirectivesImport from "npm:@unocss/transformer-directives@0.57.7";
 
 Deno.test("Unocss plugin", async (t) => {
   const site = getSite({
     src: "unocss",
   });
 
   site.use(unocss());
-  site.use(postcss());
+
+  await build(site);
+  await assertSiteSnapshot(t, site);
+});
+
+Deno.test("Unocss plugin (css file)", async (t) => {
+  const site = getSite({
+    src: "unocss",
+  });
+
+  site.use(unocss({
+    cssFile: "styles.css",
+  }));
+
+  await build(site);
+  await assertSiteSnapshot(t, site);
+});
+
+Deno.test("Unocss plugin (with transformers)", async (t) => {
+  // https://github.com/denoland/deno/issues/16458#issuecomment-1295003089
+  const transformerVariantGroup =
+    transformerVariantGroupImport as unknown as typeof transformerVariantGroupImport.default;
+  const transformerDirectives =
+    transformerDirectivesImport as unknown as typeof transformerDirectivesImport.default;
+
+  const site = getSite({
+    src: "unocss",
+  });
+
+  site.use(unocss({
+    cssFile: "styles.css",
+    cssFileTransformers: [
+      transformerVariantGroup(),
+      transformerDirectives(),
```

Comment:
I think some transformers should be enabled by default.

Reply:
Same as above (https://github.com/lumeland/lume/pull/516#discussion_r1405574946), but the transformers are much smaller than `magic-string`.